### PR TITLE
Update people

### PR DESCRIPTION
**Pull request by stefan-mueller**
- Fixes typo in @koheiw's bio
- Updates Cornelius Puschmann's affiliation
- Adjusts the spacing between boxes: If we use `</div>` five times between the first and second row, the spaces between the boxes are incoherent in the mobile version of the website, which merges the two columns into a single column.

### CODE REVIEW

**Comment by ciflikli** [comment]
@stefan-mueller removing the ```</div>``` at line 53 disturbs the grid (like you observed); further, when zoomed out/on larger screens, the first three boxes are rendered correctly however I get pushed at the end of the second row while you are placed on the third row (see image).

Making two rows seems to fix the issue (i.e. the existing code); can you provide an image of the distortion you see on mobile?

screen shot 2018-09-21 at 13 33 12

**Comment by stefan-mueller** [comment]
Below are screenshots of the website displayed on a smartphone. The vertical distance at the transition between Aki and you is around twice as large as the distances between the other QI members. Of course not a major issue, but I thought this can be fixed by adjusting the distance between the first and second row. 

problem_mobile_website

File: content/people.md [hunk 1]
```diff
@@ -50,7 +50,6 @@ id = "people"
           </div>
         </div>
       </div>   
-    </div>
```
Comment (kbenoit) [inline content/people.md]: Why is this removed - @ciflikli should it be in or not? If it should stay, please revert this line.

Comment (ciflikli) [reply content/people.md]: Yes, removing the ```</div>``` currently breaks the alignment. I am looking for a responsive bootstrap solution; in the meanwhile I will revert the line.

**Comment by kbenoit** [comment]
Great, and feel free to merge once that's done, since these are just typo changes.

**Comment by stefan-mueller** [comment]
Since the spacing issue was resolved in #27, I close this PR and delete the branch.
